src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculate = {
+    result: 0,
+
+    add(result, number) {
+      result += number;
+    },
+
+    subtract(result, number) {
+      result -= number;
```

Comment:
Similarly to the 'add' method, the 'subtract' method should update the 'result' property of the 'calculate' object. Use 'this.result' instead of the local 'result' parameter.